packages/angular/projects/clr-angular/src/layout/vertical-nav/_vertical-nav.clarity.scss

```diff
@@ -192,12 +194,19 @@
       text-align: left;
 
       .nav-group-trigger-icon {
+        //cds-icon {
+        //  @include css-var(
+        //      --color,
+        //      clr-vertical-nav-item-color
+        //      $clr-vertical-nav-item-color,
+        //      $clr-use-custom-properties
+        //  );
+        //}
         //Display
         flex-shrink: 0;
 
         //Dims
-        height: $clr-vertical-nav-item-height;
```

Comment:
Why were these sizing styles removed?

Reply:
![image](https://user-images.githubusercontent.com/15037947/97993167-5c1fb000-1dec-11eb-857c-28e14d7ff48d.png)
Take a look at the `angle`. This is how it looks like when the height and width are applied. Looks fine without them.